```
test/e2e/sequential/baseline/default_config_test.go:52:1: File is not properly formatted (gci)
	
^
test/e2e/sequential/baseline/default_config_test.go:47:106: empty-lines: extra empty line at the start of a block (revive)
var _ = ginkgo.Describe("Default configuration tests", ginkgo.Label(util.Shard0), ginkgo.Ordered, func() {
	ginkgo.BeforeAll(func() {
		util.UpdateKueueConfigurationAndRestart(ctx, k8sClient, defaultKueueCfg, kindClusterName)
	})
	
```

--- a/test/e2e/sequential/baseline/default_config_test.go
+++ b/test/e2e/sequential/baseline/default_config_test.go
@@ -45,11 +45,10 @@ const (
 )
 
 var _ = ginkgo.Describe("Default configuration tests", ginkgo.Label(util.Shard0), ginkgo.Ordered, func() {
-
 	ginkgo.BeforeAll(func() {
 		util.UpdateKueueConfigurationAndRestart(ctx, k8sClient, defaultKueueCfg, kindClusterName)
 	})
-	
+
 	ginkgo.Context("Certs", func() {
 		var (
 			ns             *corev1.Namespace

